velero/schedule/zen5-br-scripts-cm.yaml

```diff
@@ -169,6 +178,8 @@ data:
 
         #[2.2.4.3] Restore JWT configuration and keys
         info "Restore JWT configuration and keys."
+        oc patch secret -n $ZEN_NAMESPACE secret/ibm-zen-secret-jwks --patch="{\"data\": { \"jwks.json\": \"$(base64 -w0 $BACKUP_DIR/secrets/jwks/jwks.json)\" }}"
+        oc patch secret -n $ZEN_NAMESPACE secret/ibm-zen-secret-jwks --patch="{\"data\": { \"key_id\": \"$(base64 -w0 $BACKUP_DIR/secrets/jwks/key_id)\" }}"
```

Comment:
It seems that oc command will return error when we have duplicate resource type
```
✗ oc get secret -n keycloak secret/ibm-zen-secret-jwks 
error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'oc get resource/<resource_name>' instead of 'oc get resource resource/<resource_name>'

➜  oc get -n keycloak secret/ibm-zen-secret-jwks 
NAME                  TYPE     DATA   AGE
ibm-zen-secret-jwks   Opaque   2      12d

➜  oc get secret -n keycloak ibm-zen-secret-jwks 
NAME                  TYPE     DATA   AGE
ibm-zen-secret-jwks   Opaque   2      12d
```